PerimeterXModule/Properties/Settings.Designer.cs

```diff
@@ -0,0 +1,26 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
OK, I stand corrected. [According to Microsoft](https://connect.microsoft.com/VisualStudio/feedback/details/285765/resources-designer-cs-doesnt-automatically-get-rebuilt-despite-it-being-an-auto-generated-file), you do need to commit these files.